### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by ayrat555**
fix typos in comment

### CODE REVIEW

**Comment by ayrat555** [comment]
[clabot:check]

**Comment by ayrat555** [comment]
[clabot:check]

File: ethcore/evm/src/interpreter/mod.rs [hunk 1]
```diff
@@ -132,7 +131,7 @@ impl<Cost: CostType> vm::Vm for Interpreter<Cost> {
 			reader.position += 1;
 
 			// TODO: make compile-time removable if too much of a performance hit.
-			do_trace = do_trace && ext.trace_next_instruction(
+			let do_trace = ext.trace_next_instruction(
```
Comment (tomusdrw) [inline ethcore/evm/src/interpreter/mod.rs]: Actually it was used before. The point is that we stop executing `trace_next_instruction` if we receive first `false`.
Removing this condition creates unnecessary overhead of invoking tracer for every executed instruction even if tracing is disabled.

Comment (ayrat555) [reply ethcore/evm/src/interpreter/mod.rs]: But I guess this variable is set manually?

Comment (tomusdrw) [reply ethcore/evm/src/interpreter/mod.rs]: It's set manually only for the first iteration, then it's being changed with every iteration of the loop. So if `trace_next_instruction` returns `false` for the first time, next iteration of the loop will not invoke `trace_next_instruction` again (due to `do_trace &&` resolving to false immediately).

**Comment by niklasad1** [comment]
@tomusdrw You're ok with this now right?

**Pull request merged**